substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/ELFMachine.java

```diff
@@ -350,6 +397,276 @@ public int getRelocatedByteSize() {
     }
 }
 
+/**
+ * Reference: http://infocenter.arm.com/help/topic/com.arm.doc.ihi0056b/IHI0056B_aaelf64.pdf.
+ */
+enum ELFAArch64Relocation implements ELFRelocationMethod {
+    R_AARCH64_NONE(0),
+    R_AARCH64_ABS64(0x101) {
+        @Override
+        public RelocationKind getKind() {
+            return RelocationKind.DIRECT;
+        }
+
+        @Override
+        public int getRelocatedByteSize() {
```

Comment:
It looks like that `getRelocatedBytesSize` is dead. We may remove this entirely (`RelocationMethod.getRelocatedByteSize`, `RelocationSiteInfo.getRelocatedByteSize`, `ELFRelocationSection.Entry.getRelocatedByteSize` and `MachORelocationElement.getRelocatedByteSize`) We may take this opportunity and remove this code entirely since it has no meaning anyway in the common case.